policy/modules/system/init.if

```diff
@@ -2790,6 +2790,24 @@ interface(`init_read_pid_pipes',`
 	read_fifo_files_pattern($1, init_var_run_t, init_var_run_t)
 ')
 
+######################################
+## <summary>
+##	read systemd unit symlinks (usually under /run/systemd/units/)
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`init_read_units_links',`
```

Comment:
Should be `init_read_pid_symlinks()`. Slightly misleading, yes, but all the `init_var_run_t` interfaces should probably be renamed for consistency to something like `init_read_runtime_symlinks()`.